### PR TITLE
MP-9122 - Factory for converting Primitive types into Nox type "Image"

### PR DESCRIPTION
**Pull request by danield-ap**

### CODE REVIEW

File: tests/Nox.Lib.Tests/Factories/Types/NoxTypeImageFactoryTests.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using Nox.Lib.Tests.FixtureConfig;
+using Nox.Solution;
+
+namespace Nox.Lib.Tests.Factories.Types;
+
+public record ImageDto(string Url, string PrettyName, int SizeInBytes);
+public class NoxTypeImageFactoryTests
+{
+    [Theory, AutoMoqData]
+    public void CreateNoxType_FromDto_IsValid(NoxSolution noxSolution, EntityDefinitionFixture fixture)
```
Comment (dejobratic) [inline tests/Nox.Lib.Tests/Factories/Types/NoxTypeImageFactoryTests.cs:13]: Can we add a test for null a value and an invalid value?

**Review by dejobratic — CHANGES_REQUESTED** [review]
Looks good. We just need to add a few more tests.

File: tests/Nox.Lib.Tests/Factories/Types/NoxTypeImageFactoryTests.cs (hunk 1 above)
Comment (danield-ap) [reply tests/Nox.Lib.Tests/Factories/Types/NoxTypeImageFactoryTests.cs:13]: Yes, tests added.

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Review by dejobratic — APPROVED** [review]

**Review by rochar — APPROVED** [review]
